site/src/components/AvatarData/AvatarData.tsx

```diff
@@ -8,13 +8,15 @@ export interface AvatarDataProps {
   subtitle?: ReactNode;
   src?: string;
   avatar?: React.ReactNode;
+  displayTitle?: boolean;
 }
 
 export const AvatarData: FC<AvatarDataProps> = ({
   title,
   subtitle,
   src,
   avatar,
+  displayTitle = true,
```

Comment:
Perhaps there is a different `Avatar` component I should be using? I want one that accepts an empty image and creates an initialed background; however, I don't want an accompanying title and subtitle.

Reply:
you're after the regular `Avatar` component then (which is what `AvatarData` uses internally)

Reply:
Ahhh, thank you! Will remember that next time.